modules/core/src/main/java/org/apache/ignite/internal/processors/query/QueryHistoryTracker.java

```diff
@@ -114,37 +112,37 @@ else if (removeLink(node)) {
      */
     private void shrink() {
         while (true) {
-            QueryHistoryMetrics entry = evictionQueue.poll();
+            QueryHistory entry = evictionQueue.poll();
 
             if (entry == null)
                 return;
 
-            // Metrics has been changed if we can't remove metric entry.
+            // History has been changed if we can't remove history entry.
             // In this case eviction queue already offered by the entry and we don't put it back. Just try to do new
             // attempt to remove oldest entry.
-            if (qryMetrics.remove(entry.key(), entry))
+            if (qryHist.remove(entry.key(), entry))
                 return;
         }
     }
 
     /**
-     * @param node Node wchi should be unlinked from eviction queue.
+     * @param node Node which should be unlinked from eviction queue.
      * @return {@code true} If node was unlinked.
      */
-    private boolean removeLink(ConcurrentLinkedDeque8.Node<QueryHistoryMetrics> node) {
+    private boolean removeLink(ConcurrentLinkedDeque8.Node<QueryHistory> node) {
         return evictionQueue.unlinkx(node);
     }
 
     /**
-     * Gets SQL query history. Size of history could be configured via {@link
-     * IgniteConfiguration#setSqlQueryHistorySize(int)}
+     * Gets SQL query history. Size of history could be configured via
+     * {@link IgniteConfiguration#setSqlQueryHistorySize(int)}.
```

Comment:
The link doesn't refer correctly without importing class.: `import org.apache.ignite.configuration.IgniteConfiguration;`